lib/hourglass/date_time_calculations.rb

```diff
@@ -42,6 +46,21 @@ def round_interval(time_interval, project: nil)
       end
     end
 
+    def clamp?(start, project: nil)
+      clamp_limit = clamp_limit project: project
+      stop = Time.now.change(sec: 0) + 1.minute
```

Comment:
This looks like we need actual rounding here?
The +1.minute seems wrong when seconds were 0 before: "0min 0sec" => "1min 0sec".
In lib/ Hourglass::DateTimeCalculation is the "rounding" logic used for booking.

Reply:
Why are we doing rounding for sub minutes here? is it actually necessary?

Reply:
It is the behavior how time logs are updated currently. This is the reason why clamp should also use the "rounded" stop time.
Additionally we always add 1 minute to prevent a time tracked which is less then one.